test/hotspot/jtreg/compiler/lib/ir_framework/driver/TestVMProcess.java

```diff
@@ -259,7 +259,9 @@ private String getExceptionInfo() {
         int exitCode = oa.getExitValue();
         String stdErr = oa.getStderr();
         String stdOut = "";
-        if (exitCode == 134) {
+        boolean osIsWindows = System.getProperty("os.name").toLowerCase().contains("windows");
```

Comment:
You can use `Platform.isWindows()` instead.

Reply:
Better, thanks. And replaced. (My implementation was actually pretty similar to `Platform.isWindows()`, there seems not to be a much nicer way, but only name matching.)